src/integ_test_resources/flutter/amplify/cloudformation_template.yaml

```diff
@@ -0,0 +1,61 @@
+Parameters:
+  FullRepoName:
+    Type: String
+    Description: Example "aws-amplify/amplify-flutter"
+    AllowedPattern: ^[a-zA-Z0-9-\_\.]+\/[a-zA-Z0-9-\_\.]+$
+    ConstraintDescription: must contain two alphanumeric strings with dashes, periods, or underscores separated by a slash
+    
+Resources:
+  Role:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: pull_amplify_integration_test_configs
+      Policies:
+        - PolicyName: AmplifyPullIntegrationTestsS3
+          PolicyDocument:
+            Statement:
+              -
+                Effect: Allow
+                Action:
+                  - s3:GetObject
+                  - s3:ListBucket
+                Resource: "arn:aws:s3:::amplify-*"
```

Comment:
There is no way to scope this down?

Reply:
I scoped this down in latest revision by doing 2 things:
1) require an env name for all the amplify envs used and limit scope to that (flutter will use "test" when run)
2) allow the name of an amplify app name for the case where someone wants to use this for a single app and limited scope when that parameter is provided.

I tested both cases and got expected failures for case 2 when trying to pull for some other app.